nasa/earthdata-search
# EDSC-3358: Adding related collections to the collection details page

# Overview

### What is the feature?

Adding related collections and metrics events to the collection details page

### What is the Solution?

Display top 3 related collections on the page

### What areas of the application does this impact?

Collection details page

# Testing

### Reproduction steps

- **Environment for testing:** ANY
- **Collection to test with:** ANY with related collections

1. View collection
2. Click related collection
3. Confirm granules page is displayed
4. Confirm metrics events fire

# Checklist
